next/components/organisms/NavBar/NavMenu/MobileNavMenuContent.tsx

```diff
@@ -30,14 +31,13 @@ const MobileNavMenuContent = ({ menuItem, colorStyle }: NavMenuContentProps) =>
       {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
       <ul className="flex w-full flex-col gap-2 px-4 py-8">
         <li className="w-full">
-          <button
-            type="button"
-            onClick={() => setMenuValue('')}
+          <Button
+            onPress={() => setMenuValue('')}
             className="-my-2 flex w-full items-center gap-2 py-2"
             aria-label={t('aria.backTo', { backTo: parentLabel })}
           >
             <ChevronLeftIcon aria-hidden /> {parentLabel}
```

Comment:
Could this icon be used as `startIcon`?

Reply:
yes.. that one is one of the first ones I did. forgot about "icon" props